crates/examples/Cargo.toml

```diff
@@ -7,3 +7,7 @@ edition = "2021"
 
 [dependencies]
 configuration = { path = "../configuration" }
+orchestrator = { path = "../orchestrator" }
+provider = { path = "../provider" }
+support = { path = "../support" }
```

Comment:
Support types needed by external users should be re-exported by the orchestrator.

Reply:
Maybe having a wrapper zombienet crate organizing all re-exports and features could be good, wdyt ?